benchmark/README.md

```diff
@@ -55,8 +55,8 @@ In the event you get a message that you need to select a CRAN mirror first.
 
 You can specify a mirror by adding in the repo parameter.
 
-If we used the "http://cran.us.r-project.org" mirror, it could look somehting like
-this:
+If we used the "http://cran.us.r-project.org" mirror, it could look something
```

Comment:
I believe this will be covered by https://github.com/nodejs/node/pull/11189 once that lands.